env-techsalesdraft.yml

```diff
@@ -2,7 +2,7 @@
 ###################################### BASIC CONFIGURATION ######################################
 #################################################################################################
 
-contact: support.cloud.devops@aveva.com # used for tagging and sending licences (see documentation)
+contact: tanytest@aveva.com # used for tagging and sending licences (see documentation)
```

Comment:
This looks fine please add more comments under tagging

Reply:
Issue resolved